woocommerce/woocommerce
# Add WooCommerce Services Banner to Extensions Pages - Take 2

This PR implements @olaolusoga's designs from p7bbVw-1tf-p2 (under the 'Where else?' section). cc @jeffstieler @nabsul

Here's what it looks like on the featured page:

<img width="1115" alt="screen shot 2017-05-30 at 11 55 42 pm" src="https://cloud.githubusercontent.com/assets/11487924/26615265/986acd54-4593-11e7-934b-b5c5f4d5544f.png">

The content (title, description, image) on the featured page can still be changed using the featured json file, but has the defaults shown above.

On the shipping page:

<img width="1118" alt="screen shot 2017-05-30 at 11 55 52 pm" src="https://cloud.githubusercontent.com/assets/11487924/26615255/907c87c2-4593-11e7-8d53-0268574072e5.png">

# Testing instructions
1. In /includes/admin/class-wc-admin-addons.php make sure to use the test json files (I hosted the files locally):
On line 27 use this [featured json file](http://cld.wthms.co/um7Np)
The banner is hardcoded on the shipping pages, so no more changes are required
2. Make sure to delete your WordPress transients before testing, you can use a plugin like [Transient Cleaner](https://wordpress.org/plugins/artiss-transient-cleaner/)
3. Go to `wp-admin/admin.php?page=wc-addons&section=featured` and `wp-admin/admin.php?page=wc-addons&section=shipping_methods` to see the changes

# Still to do
- [ ] get final boxes image from @olaolusoga 
- [ ] better responsive designs?
- [x] handle if plugin has been installed but not activated (currently runs into an error when using the plugin install URL)
- [x] potentially use background installer, cc @jeffstieler
